src/locales/keys.ts

```diff
@@ -0,0 +1,70 @@
+export enum Keys {
+  TITLE = 'TITLE',
```

Comment:
Går det an å få til keys uten å duplisere KEY_NAME = 'KEY_NAME'?
Kanskje flytte denne til egen fil dersom den blir lang?

Reply:
Skal gå helt fin, det.
`export enum Keys {
  TITLE,
  DEN_ANSATTE
}`

Reply:
Det vil ikke virke grunnet måten det er satt opp for type safety